main.py

```diff
@@ -1595,21 +1595,48 @@ def get_smmry(message):
 
 async def whoami(client, message):
     """
-    Handles !whoami
+    Handles !me
 
     :param client: The Client
     :param message: The message
     :return: None
     """
     author = str(message.author.display_name)
     if author in users and users[author] != {}:
-        await client.send_message(message.channel,
-                                  "Well, I don't know you that well, but "
-                                  "from what I've been hearing on the "
-                                  "streets...")
+        message_output = "Well, I don't know you that well, but " \
+                         "from what I've been hearing on the " \
+                         "streets...\n"
+
         for k, v in users[author].items():
-            await client.send_message(message.channel,
-                                      "Your {} is {}.".format(k, v))
+            if k == "duel_record":
+                if v[0] < 10:
+                    output = "You're a pretty green dueler"
+                elif v[0] < 100:
+                    output = "You're a seasoned dueler"
+                else:
```

Comment:
If we are doing seasons and frequent record resets, Are we going to want to set the master bad this high?